src/modules/loadCharacters.js

```diff
@@ -67,32 +100,32 @@ export async function fetchCharacterDetails(characterId) {
 export function updateModalContent(character) {
   const modalBody = document.querySelector('.modal-body');
   if (!modalBody) {
-    // console.error('Modal body not found');
-    return;
+    throw new Error('Modal body not found');
+    // return;
   }
 
   const characterContainer = modalBody.querySelector('.character-content');
   if (!characterContainer) {
-    // console.error('Character container not found');
-    return;
+    throw new Error('Character container not found');
+    // return;
   }
 
   const features1 = modalBody.querySelector('.features-1');
   if (!features1) {
-    // console.error('Features 1 container not found');
-    return;
+    throw new Error('Features 1 container not found');
+    // return;
   }
 
   const features2 = modalBody.querySelector('.features-2');
   if (!features2) {
-    // console.error('Features 2 container not found');
-    return;
+    throw new Error('Features 2 container not found');
+    // return;
   }
 
   const commentsList = modalBody.querySelector('ul'); // Directly target the <ul> element
   if (!commentsList) {
-    // console.error('Comments list not found');
-    return;
+    throw new Error('Comments list not found');
+    // return;
   }
```

Comment:
Great use of Github flows 👍
No linters errors 👌

**Requested changes**
It would make you code more readable and great looking when you reduce the highlighted word. Just delete the //Directly target the <ul> element and you are good to go.

Reply:
Those are comments to make anyone reading the code to understand it better

Reply:
I will remove some of the comments when finalizing the project

Reply:
Ok great. Your code is good.